doc/sfp-cmis/Interface-Link-bring-up-sequence.md

```diff
@@ -149,11 +151,18 @@ Please refer to the  flow/sequence diagrams which covers the following required
 # Feature enablement
   This feature (optics Interface Link bring-up sequence) would be enabled on per platform basis.
   There could be cases where vendor(s)/platform(s) may take time to shift from existing codebase to the model (work-flows) described in this document.
-  In order to avoid any breakage and ensure gradual migration of different platforms/vendors to this model, there would be new field (flag) in xcvrd to enable/disable this feature. 
-  When xcvrd spawns on LC/board, it would invoke platform plugin to check with the platform (hwsku) whether this feature is yet supported on underlying platform (board/LC) or not
+  In order to avoid any breakage and ensure gradual migration of different platforms/vendors to this model, will add this new workflow to enable/disable this feature:
+  
+  In order to enable this feature, the platform would set ‘skip_xcvrd_cmis_mgr’ to ‘false’ in their respective pmon_daemon_control.json as part of platform bootstrap. When xcvrd would spawn on that hwsku (LC/board), it would parse ‘skip_xcvrd_cmis_mgr’ and if found 'false', it would launch CMIS task manager. This implies enabling this feature. 
+
+Else, if ‘skip_xcvrd_cmis_mgr’ is set/found 'true' by xcvrd, it would skip launching CMIS task manager and this feature would remain disabled.
+If a platform/vendor does not specify/set ‘skip_xcvrd_cmis_mgr’, xcvrd would exercise the default workflow (i.e. when xcvrd detects QSFP-DD, it would luanch CMIS task manager and initialize the module per CMIS specification). 
```

Comment:
I would suggest the default behavior is **NOT** to start CMIS task manager, which means that if 'skip_xcvrd_cmis_mgr' is not defined, then xcvrd will NOT start CMIS task manager.  If not, all the vendors have to go through all their platforms and add an explicit definition for their legacy platforms which not support this feature.

Reply:
@keboliu Majority of the vendor platform are going to use CMIS manager, if not can you list the platforms which are NOT going to use CMIS manager that way we can decide what should be the default behaviour.

Reply:
@prgeor I will raise a PR to add the configuration for Nvidia platforms to disable the CMIS tasks.

Reply:
@keboliu Thanks